### ISSUE TITLE
CREATE EXTENSION gp_internal_tools change the param search_path

### ISSUE THREAD

**Issue by LittleWuCoding** [greenplum-db/gpdb#10575]
### Greenplum version or build
greenplum 6.4
### OS version and uname -a
rhel7.5
### Actual behavior
```
postgres=# show search_path ;
  search_path   
----------------
 "$user",public
(1 row)

postgres=# CREATE EXTENSION gp_internal_tools;
CREATE EXTENSION
postgres=# show search_path ;
  search_path  
---------------
 session_state
(1 row)
```
you can see that the search_path was changed, which may get users confused(eg. can not select from a table in public without specify prefix public after creating the extension in the same psql command line). 

I found that the rerason is that in /gpcontrib/gp_internal_tools/gp_internal_tools--1.0.0.sql:
```
CREATE SCHEMA session_state;
SET search_path = session_state;
....
....
```
which omission the follow sql statement at the end:
```
SET search_path TO default;
```
also ,there are typos in /gpcontrib/gp_internal_tools/gp_internal_tools--1.0.0.sql:
```
-- complain if script is sourced in psql, rather than via CREATE EXTENSION
\echo Use "CREATE EXTENSION file_fdw" to load this file. \quit
```
the extension name in the second line should be **_gp_internal_tools_** but not **_file_fdw_** ~

**Comment by LittleWuCoding** [comment]
Also ,there is also problems when DROP EXTENSION, which related to /gpcontrib/uninstall_gp_session_state.sql:
```
SET search_path = session_state;

BEGIN;

DROP VIEW session_level_memory_consumption;
DROP FUNCTION session_state_memory_entries_f_on_master();
DROP FUNCTION session_state_memory_entries_f_on_segments();
DROP SCHEMA session_state;

COMMIT;
```
there also should reset the search_path for that after drop extension, the search_path‘s  value is 'session_state', but there is no more schema session_state in database which resulting  bad results, for example
```
postgres=# drop extension gp_internal_tools cascade;
NOTICE:  drop cascades to table session_state.t1
DROP EXTENSION
postgres=# show search_path ;
  search_path  
---------------
 session_state
(1 row)

postgres=# CREATE EXTENSION gp_internal_tools;
ERROR:  no schema has been selected to create in  (seg0 10.47.242.100:14000 pid=26072)
```

**Comment by JunfengYang** [comment]
@LittleWuCoding, good cache! Would you like to open a PR to fix it?

**Comment by LittleWuCoding** [comment]
> @LittleWuCoding, good cache! Would you like to open a PR to fix it?

OK, I will do a PR it  by this weekend~

**Comment by LittleWuCoding** [comment]
My PR:  #10649

**Comment by JunfengYang** [comment]
Fix commit `9d9662f0bf217ed40b25f6e6cba01f1308a56126` on Master and `2fbf4808a595e654e3fa01b93bbf474c23ba8e81` on 6X. So close it.
